jina/serve/runtimes/request_handlers/data_request_handler.py

```diff
@@ -62,18 +62,18 @@ def _load_executor(self, metrics_registry: Optional['CollectorRegistry'] = None)
                 extra_search_paths=self.args.extra_search_paths,
             )
 
-        except BadConfigSource as ex:
+        except BadConfigSource:
             self.logger.error(
                 f'fail to load config from {self.args.uses}, if you are using docker image for --uses, '
                 f'please use "docker://YOUR_IMAGE_NAME"'
             )
-            raise ExecutorFailToLoad from ex
-        except FileNotFoundError as ex:
+            raise
+        except FileNotFoundError:
             self.logger.error(f'fail to load file dependency')
-            raise ExecutorFailToLoad from ex
-        except Exception as ex:
+            raise
+        except Exception:
             self.logger.critical(f'can not load the executor from {self.args.uses}')
-            raise ExecutorFailToLoad from ex
+            raise
```

Comment:
I opted to remove our ExecutorFailToLoad exception for all cases, not just the internal error one.
The trade-off is the following:
- Con: We are not explicit anymore about it being the _Executor_ that fails
- Pro: The user only sees the actual problem and where to fix it

IMO it will be implicitly obvious that it is the executor that fails, so i think it makes sense to have it this way. @jacobowitz lmk if you agree or not